src/ars/core/repository.py

```diff
@@ -192,13 +193,156 @@ async def get(
 
         return requests
 
-    async def update(  # noqa: C901
+    def handle_generic_errors(
```

Comment:
Make internal methods private by starting them with an underscore.

Reply:
Here it is unclear whether these "handle_xxx" methods are used in the context or a create or an update or both. Should be understandable from the name, like `handle_validity_period_update`.